src/main/java/jenkins/plugins/gerrit/workflow/GerritReviewStep.java

```diff
@@ -64,10 +68,27 @@ protected Void run() throws Exception {
           int changeId = Integer.parseInt(matcher.group("changeId"));
           int revision = Integer.parseInt(matcher.group("revision"));
           listener.getLogger().format("Gerrit review change %d/%d label %s=%d (%s)%n", changeId, revision, label, score, message);
-          ReviewInput reviewInput = new ReviewInput().label(label, score).message(message);
+          ReviewInput reviewInput = new ReviewInput().message(message);
+          boolean notifyOwner = false;
+          if (labels == null) {
+            if (label != null) {
+                reviewInput.label(label, score);
+                if (score < 0) {
+                  notifyOwner = true;
+                }
+            }
+          }
+          else {
+            for (Map.Entry<String, Integer> l : labels.entrySet()) {
```

Comment:
If you follow my suggestion above, you don't have an if/then/else anymore.

Reply:
See this... I tried... and I still have... as I cannot add label/score into an entry as one can be gotten without the other.